edx_notifications/server/web/utils.py

```diff
@@ -40,10 +40,16 @@ def get_notifications_widget_context():
             ). format(base_url=reverse('edx_notifications.consumer.notifications')),
             'renderer_templates_urls': reverse('edx_notifications.consumer.renderers.templates'),
         },
+        'global_variables': {
+            'app_name': 'Your App Name Here',
+        },
         'view_templates': {
             'notification_icon': get_template_path('notification_icon.html'),
             'notification_pane': get_template_path('notification_pane.html'),
         }
     }
 
+    if override_context:
+        context.update(override_context)
```

Comment:
This is where an application can pass in additional context, for example, the app_name which only the application tier knows about.